opentelemetry/proto/collector/logs/v1/logs_service.proto

```diff
@@ -43,4 +43,33 @@ message ExportLogsServiceRequest {
 }
 
 message ExportLogsServiceResponse {
+  // The details of a partially successful export request.
+  //
+  // If the request is only partially accepted
+  // (i.e. when the server accepts only parts of the data and rejects the rest)
+  // the server MUST initialize the `partial_success` field and MUST
+  // set the `rejected_<signal>` with the number of items it rejected.
+  //
+  // Servers MAY also make use of the `partial_success` field to convey
+  // warnings/suggestions to senders even when the request was fully accepted.
+  // In such cases, the `rejected_<signal>` MUST have a value of `0` and
+  // the `error_message` MUST be non-empty.
+  ExportLogsPartialSuccess partial_success = 1;
```

Comment:
Please explain if there is a difference between `ExportLogsPartialSuccess` not present (proto supports not present for messages) and an empty `ExportLogsPartialSuccess` (where rejected is 0 and error_message is empty).

Reply:
We debated whether to make this field optional (at length). The protocol compiler defines this equivalence, between non-optional-field-not-present and field-present-with-all-zeros. I feel like we answered your question by deciding to make this message not optional, which leaves me unsure what else needs to be written here. Can you suggest an edit?

Reply:
Done in [49bcec0](https://github.com/open-telemetry/opentelemetry-proto/pull/414/commits/49bcec06f20a67f28b77894ebd23591b2432360f). Let me know if this clarify things!

Reply:
@joaopgrassi added an explicit statement (in three places). @bogdandrutu does this look good?

Reply:
This looks good.

@jmacd "optional" has no effect on "messages" which by default are "optional" and cannot be changed (unless you use a different version of the compiler like the gogo proto). Optional keyword is important only for primitives.

Reply:
@bogdandrutu Thanks. I had misunderstood the rule for optional with message fields.  In fact, I stated the opposite during a SIG call in favor of counting `rejected_<thing>` instead of `accepted_<thing>`, because it lets the zero-state match legacy behavior. It also means we can ignore field presence (and/or means we can continue to use Gogo).

Given this information, I still prefer to count rejected things. @bogdandrutu @joaopgrassi does this change your opinion of this PR and https://github.com/open-telemetry/opentelemetry-specification/pull/2696?

Reply:
@jmacd in my view we are good, I also like the rejected approach. 

I think with the instructions we have in the spec + the extra text I added as requested by @bogdandrutu make things clear.

Pointing out that the "empty" value of `partial_success` is equivalent to it not being set and that is also equivalent of a "total success" makes sense to me.